aws/infra_configs/cluster_config.yaml

```diff
@@ -0,0 +1,44 @@
+# For details, Please check eksctl documentation or API specs.
```

Comment:
I believe this file will not be needed for kustomize. 
@surajkota  Can you confirm ?